src/world/area_sbk/sbk_01.c

```diff
@@ -1,5 +1,6 @@
 #include "common.h"
 
+//#include "world/common/SpawnSunEffect.c"
```

Comment:
?

Reply:
I can't add this yet due to files not being split yet. This'll be fixed today